directives/annotated.py

```diff
@@ -153,10 +174,13 @@ def run(self):
         env.annotated_annotation_count += 1
         node['annotation-number'] = env.annotated_annotation_count
         node['name-of-annotated-section'] = env.annotated_name
+        if self.arguments:
+            node['replacement'] = self.arguments[0]
         return [node]
 
 def visit_annotation_node(self, node):
-    self.body.append('<div class="container codecomment comment-%s-%s">' % (node['name-of-annotated-section'], node['annotation-number']))
+    replacement_attrib = ' replacement="' + node['replacement'] + '"' if 'replacement' in node else ""
```

Comment:
Seems like there is an HTML attribute `replacement` here. It is not a standard attribute, so it should be `data-replacement`. Data attributes are intended for custom data storage. (This is only about the HTML: Sphinx nodes may use whatever attributes.)